src/main/java/com/epam/aidial/core/service/ResourceService.java

```diff
@@ -229,7 +229,9 @@ public void computeResource(ResourceDescription descriptor, Function<String, Str
         try (var ignore = lockService.lock(redisKey)) {
             String body = getResource(descriptor, false);
             String updatedBody = fn.apply(body);
-            putResource(descriptor, updatedBody, true, false);
+            if (updatedBody != null) {
```

Comment:
we do we not update a resource with nullable body?

Reply:
putResource expect non-null result